backend/stakepoold/rpcclient.go

```diff
@@ -224,3 +224,50 @@ func walletFetchUserTickets(ctx *appContext) map[string]UserTickets {
 
 	return userTickets
 }
+
+func walletFetchUserTicketsNew(ctx *appContext) map[string]UserTickets {
+	userTickets := map[string]UserTickets{}
+
+	ticketcount := 0
+	usercount := 0
+	for msa := range ctx.userVotingConfig {
+		addr, err := dcrutil.DecodeAddress(msa, ctx.params)
+		if err != nil {
+			log.Infof("unable to decode multisig address: %v", err)
+			continue
+		}
+
+		spui, err := ctx.walletConnection.StakePoolUserInfo(addr)
+		if err != nil {
+			log.Errorf("unable to fetch tickets for userid %v multisigaddr %v: %v",
+				ctx.userVotingConfig[msa].Userid, msa, err)
+			continue
+		}
+		if spui != nil && len(spui.Tickets) > 0 {
+			for _, ticket := range spui.Tickets {
+				switch ticket.Status {
+				case "live":
+					hash, err := chainhash.NewHashFromStr(ticket.Ticket)
+					if err != nil {
+						log.Infof("invalid ticket %v", err)
+						continue
+					}
```

Comment:
We should just skip this validation.  If wallet is handing out invalid ticket hashes then there are big wallet problems going on.  And the ticket hashes won't get called to vote anyway.